step.yml

```diff
@@ -53,43 +53,45 @@ inputs:
         **The path should be relative** to the repository root. For example, `./gradlew`,
         or if it is in a sub directory, `./sub/dir/gradlew`.
       is_required: true
-  - apk_file_include_filter: "*.apk"
+  - app_file_include_filter: |
+      *.apk
+      *.aab
     opts:
       category: Export Config
-      title: "APK file include filter"
+      title: "APK and AAB file include filter"
       description: |
-        The step will copy the generated APK files that match this filter into the Bitrise Deploy Directory.
+        The step will copy the generated APK and AAB files that match this filter into the Bitrise Deploy Directory.
 
         This filter is a standard find commands -path pattern, see: http://linux.die.net/man/1/find
 
-        If you specify an empty filter, the step will set this filter to: `*.apk`.
+        If you specify an empty filter, the step will set this filter to: `*.apk *.aab`.
 
         Example:
 
-        Copy every APK file:
+        Copy every AAB file:
 
         ```
-        "*.apk"
+        "*.aab"
         ```
 
         Copy every APK file with a filename that contains `release`, like (`./app/build/outputs/apk/app-release-unsigned.apk`):
 
         ```
         *release*.apk
         ```
-  - apk_file_exclude_filter: |
+  - app_file_exclude_filter: |
```

Comment:
Do not remove the `apk_file_exclude_filter`.
The input is deprecated, so put it under the `Deprecated` category, but the step should be backward compatible.

Reply:
If you remove the input from the step.yml, the CLI will not parse the input, even if it's provided to the step.